pkg/kubernetes/node/v1alpha1/kubernetes_test.go

```diff
@@ -0,0 +1,106 @@
+package v1alpha1
+
+import (
+	"errors"
+	"reflect"
+	"testing"
+
+	v1 "k8s.io/api/core/v1"
```

Comment:
nit : `corev1` as we are using the same alias in other places too ?

Reply:
Done